docs: update appendix hardware db sizing (#1231)
bareos/bareos: docs/manuals/source/Appendix/HardwareSizing.rst

Typically this is about **90-95%** of the database's total size.
The **size of the file table** depends on the number of files that are stored and the average length of a filename (without path).

Roughly 40% of the file table size is consumed by its **indexes**.

> The other noticeable table is **path** with its indexes. The amount of space used is roughly :math:`\frac{1}{10}` of **file** table.

Author: ```suggestion
The other noticeable table is **path** with its indexes. The amount of space used is roughly :math:`\frac{1}{10}` of the **file** table.
```

To **calculate the number of files** in the DB, the number of files being backed up from all systems needs to be multiplied by the number of times that they will be kept in the database.

> The **amount of data per file or path** in the DB, depends on the size of filenames or path length that are being backed up, but we have analyzed some real-world examples and found that values between **250 and 350 bytes per row** are usual.

Reviewer: Is that true? Is that also the value for path entries? Usually I would expect the path entries to be much longer than the file entries?

Author: Check real customer report in task OP #5209 section file

The size of the path indexes is around 60% of the table size.

So the calculation of the size of the file and path tables can be approximated with the following formula:

.. math::
   \begin{split}
   s &= n_f \times n_b \times 300 \frac{\mbox{bytes}}{\mbox{row}} \\
   p &= \frac{s}{10} \\
   s &: \mbox{storage required for file table} \\
   n_f &: \mbox{number of files in a (full) backup} \\
   n_b &: \mbox{number of (full) backups} \\
   p &: \mbox{storage required for path table} \\
   \end{split}

*Example:* If **2.000.000 files** are backed up during a full backup, a full backup is run **every week** and the retention of the backups is **4 weeks**, the total amount of files would be

.. math::
   \begin{split}

>    n_f &= 2.000.000\ \mbox{Files} \\

Reviewer: Have you checked that these numbers are true for current databases? (Number of path, file entries compared to total db size in real installations)?

Author: Here, we propose some mathematical way to approximate the size of the database. Again the formula can be checked against the real world example we got from support and bug entries.

   n_b &= 4\ \mbox{Full Backups} \\
   s &= n_f \times n_b \times 300 \frac{\mbox{bytes}}{\mbox{row}} \\
     &= 2.000.000\ \mbox{Files} \times 4\ \mbox{Full Backups} \times 300 \frac{\mbox{bytes}}{\mbox{row}} \\
     &= 2.400.000.000\ \mbox{bytes} \\

>      &= 2.400\ \mbox{MB} \\

Reviewer: ```suggestion
     &= 2.4\ \mbox{GB} \\
```

     \ \ \\
   p &= \frac{\mbox{s}}{10} \\
     &= \frac{2.400}{10}\ \mbox{MB} \\
     &= 240\ \mbox{MB} \\
   \end{split}

> About 40% of the File and 60% of path table size should be available as RAM, so about 1104 MB.

Reviewer: ```suggestion
About 40% of the File and 60% of path table size should be available as RAM, so about 1GB.
```
